src/Fantomas.Core.Tests/TestHelpers.fsi

```diff
@@ -9,7 +9,10 @@ module String =
 
 val config: FormatConfig
 val newline: string
-val formatSourceString: isFsiFile: bool -> s: string -> config: FormatConfig -> string
+
+val formatSignatureString: (string -> FormatConfig -> string)
+val formatSourceString: (string -> FormatConfig -> string)
```

Comment:
Nit: maybe a `///` to mention that the code will be an implementation file, and users should use `formatSignatureString` if they want to format a signature.